index.js

```diff
@@ -189,6 +189,34 @@ inputTitle.value = ``;
 
 }
 
+// ************************ Code Creating Static once clicked and on a setTimeout *********************** //
+
+//selecting the image
+let mainElement = document.querySelector("img");
+// picking up the p tag
+let list = document.querySelector("p")
+// two event handlers which will handle the click on the shaking tele.
+mainElement.addEventListener("click", clickStatic)
+list.addEventListener("click", clickStatic)
+
+function clickStatic() {
+//   console.log(setTimeout(myTimeout))
+  mainElement.style.display="none";
+  list.style.display="none";
+  let errorLostScreen = document.querySelector(".tv")
+//   console.log(errorLostScreen)
+  errorLostScreen.classList.remove("no-signal");
+  errorLostScreen.classList.add("tv-static")
+  setTimeout(function() {
+    errorLostScreen.classList.remove("tv-static");
+    mainElement.style.display = "block";
+    list.style.display = "block";
+  }, 1500);
+//   console.log('yo,yo,yo')
+  return errorLostScreen
```

Comment:
What does this return actually do? This is the only bit I don't quite get what's happening. Will it not work without this? 

Whole function is super clear to follow btw! 😸

Reply:
Thank you so much ⭐ 

I'll be honest I did not try this without the return 😆 good call on this 💯